lib/iris/tests/__init__.py

```diff
@@ -441,10 +449,21 @@ def assertRepr(self, obj, reference_filename):
         self.assertString(repr(obj), reference_filename)
 
     def _check_same(self, item, reference_path, type_comparison_name='CML'):
+        def _patch_history_attr(cdl):
+            result = iso8601.search(cdl)
+            if result:
```

Comment:
Ahh, okay I understand.

I'd argue that there is no need to strip **all** the time stamps from the `history` attribute, only the first one, because I believe an application would only prefix one entry to the attribute. However, it would be possible to strip out all the time stamps with a little more effort. Selfishly for my use case, that's all I want to do, so that's what I've done :wink: